packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenu.md

```diff
@@ -44,26 +43,22 @@ class SingleOption extends React.Component {
         });
       };
 
-      this.toggleTemplate = ({toggleTemplateProps}) => {
-        const { text } = toggleTemplateProps;
-        return <React.Fragment>{text}</React.Fragment>
-      };
+      this.toggleTemplate = <React.Fragment>{this.state.toggleTemplateText}</React.Fragment>;
```

Comment:
is the React.Fragment needed? Can you pass a string into the `toggleTemplate` prop?